app/controllers/patients_controller.rb

```diff
@@ -93,6 +94,7 @@ def new_group_member
     dashboard_crumb(params.permit(:nav)[:nav], parent)
 
     @patient = Patient.new(parent.attributes.slice(*group_member_subset.map(&:to_s)))
+    @patient.contact_type = 'Unknown'
```

Comment:
Should not be needed since this defaults to `Unknown`.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1116/commits/b0a66db3ecaa1af2ddd97ffc29f20efdf5ca7b68